src/components/hero.scss

```diff
@@ -95,22 +96,47 @@
         margin-bottom: 35px;
       }
     }
+  }
+}
 
```

Comment:
question: why is the button smaller on mobile? It doesn't need to, right? 
![Uploading image.png…]()